mlflow/pyspark/ml/__init__.py

```diff
@@ -216,6 +448,26 @@ def autolog(
           supported.
           See ``log_models`` param below for details.
 
+    **How does autologging work for meta estimators?**
+          When a meta estimator (e.g. `pyspark.ml.Pipeline`_, `pyspark.ml.tuning.CrossValidator`_)
```

Comment:
To insert a link for `pyspark.ml.Pipeline`, we need to define the link target like this:

https://github.com/mlflow/mlflow/blob/9d9d4b1f1f62de82637e24c1eb1daeec405e6c30/mlflow/sklearn/__init__.py#L565-L580